sdk/containerregistry/azure-containerregistry/samples/README.md

```diff
@@ -10,7 +10,7 @@ products:
 urlFragment: containerregistry-samples
 ---
 
-These code samples show common scenario operations with the Azure Container Registry client library.
+These code samples show common scenario operations with the Azure Container Registry client library. The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` set including the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).
```

Comment:
Not sure if this is just me or not but I think this sentence should be like:
```
The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` is set, which includes
```

Reply:
Updated my poor grammar, thanks Catalina!